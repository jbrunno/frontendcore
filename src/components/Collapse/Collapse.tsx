import React from 'react';
import {
  Collapse as MuiCollapse,
  CollapseProps as MuiCollapseProps,
} from '@mui/material';

export type CollapseProps = MuiCollapseProps;

export function Collapse(props: CollapseProps) {
  return <MuiCollapse {...props} />;
}
export default Collapse;
