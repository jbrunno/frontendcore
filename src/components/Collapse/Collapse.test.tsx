import React from 'react';
import { render } from '@test-utils/component-testing';
import Collapse from './Collapse';

test('renders the Collapse', () => {
  const { container } = render(<Collapse />);
  expect(container).toMatchSnapshot();
});
