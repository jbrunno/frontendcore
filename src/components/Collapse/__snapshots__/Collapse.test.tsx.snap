// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Collapse 1`] = `
.emotion-0 {
  height: 0;
  overflow: hidden;
  -webkit-transition: height 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: height 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  visibility: hidden;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

.emotion-2 {
  width: 100%;
}

<div>
  <div
    class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden emotion-0"
    style="min-height: 0px;"
  >
    <div
      class="MuiCollapse-wrapper MuiCollapse-vertical emotion-1"
    >
      <div
        class="MuiCollapse-wrapperInner MuiCollapse-vertical emotion-2"
      />
    </div>
  </div>
</div>
`;
